crochet/tests/test_shutdown.py

```diff
@@ -69,7 +68,12 @@ class FakeThread:
             def is_alive(self):
                 return alive
 
-        w = Watchdog(FakeThread(), lambda: done.append(True))
```

Comment:
I feel like this test could be simplified to not even have a thread, and just have a flag that gets sets to True... so correspondingly the docstring should change.